utils/s3/client.cc

```diff
@@ -765,7 +765,11 @@ future<> client::multipart_upload::abort_upload() {
     s3l.trace("DELETE upload {}", _upload_id);
     auto req = http::request::make("DELETE", _client->_host, _object_name);
     req.query_parameters["uploadId"] = std::exchange(_upload_id, ""); // now upload_started() returns false
-    co_await _client->make_request(std::move(req), ignore_reply, http::reply::status_type::no_content);
+    co_await _client->make_request(std::move(req), ignore_reply, http::reply::status_type::no_content).handle_exception([](const auto&) -> future<> {
```

Comment:
might want to use `std::exception_ptr` instead of `auto` to be more specific
and avoid template instantiation, which improves build-time performance. because `[](const auto&) ...` actually creates a templated lambda.